sky/templates/kubernetes-ray.yml.j2

```diff
@@ -25,6 +28,18 @@ provider:
 
     ssh_jump_image: {{k8s_ssh_jump_image}}
 
+    ingress_name: {{cluster_name_on_cloud}}-skypilot-ingress
+
+    {%- if ports is not none %}
```

Comment:
Current ports implementation won't put ports in the provider config. Instead, we use `handle.launched_resources.ports` to keep track of all opened ports. Reference: https://github.com/skypilot-org/skypilot/blob/bf33602836372dd70c2442c39d92e0cc27fa0e5b/sky/backends/cloud_vm_ray_backend.py#L3067-L3077
Can we change to the same implementation logic to support `sky launch --ports <new-ports>` to open new ports on an existing cluster? It seems like we can construct all other fields using `port` and `cluster_name` so keeping track of ports should be enough.

Reply:
Addressed in https://github.com/skypilot-org/skypilot/pull/2713/commits/186193cf7e82f890dbe6a704d644e48b1e63a788